pages/article.js

```diff
@@ -0,0 +1,141 @@
+import Arrow from '../public/img/arrow.svg';
+
+const Article = () => {
+  const [isDetailsOpen, setIsDetailsOpen] = useState(false);
+
+  const openDetailsHandler = () => {
+    setIsDetailsOpen(!isDetailsOpen);
+  };
+
+  return (
+    <LayoutSingle>
+      <h1 className="headline--1 underline-large">
+        Muzeum trati a Ostře sledovaných vlaků
+      </h1>
+      <div className="flex flex-col-reverse md:flex-row">
+        <div className="md:w-[70%] md:mr-[5%]">
+          <p className="paragraph">
+            Lipsum dolor sit amet, consectetur adipiscing elit. Praesent
+            egestas, augue elementum elementum facilisis, ligula velit fermentum
+            libero, dictum malesuada justo dui sed sapien. Quisque ac erat
+            scelerisque enim mattis viverra. Pellentesque pulvinar ex gravida,
+            consectetur erat et, semper ligula. Aenean scelerisque urna augue,
+            ut accumsan enim bibendum ut. Quisque dictum felis faucibus erat
+            consequat, quis ullamcorper orci faucibus. Fusce gravida urna dolor,
+            sit amet ultricies leo posuere eu. Quisque augue purus, porttitor
+            quis sagittis vitae, cursus vel turpis.
+          </p>
+          <h2 className="headline--2 border-b-2">Lodenicke nadrazi</h2>
+          <p className="paragraph">
+            Quisque ac erat scelerisque enim mattis viverra. Pellentesque
+            pulvinar ex gravida, consectetur erat et, semper ligula. Aenean
+            scelerisque urna augue, ut accumsan enim bibendum ut. Quisque dictum
+            felis faucibus erat consequat.
+          </p>
+
+          <h3 className="headline--3 border-b-2">Obecni urad</h3>
+          <p className="paragraph">
+            Aenean scelerisque urna augue, ut accumsan enim bibendum ut. Quisque
+            dictum felis faucibus erat consequat, quis ullamcorper orci
+            faucibus. Fusce gravida urna dolor, sit amet ultricies leo posuere
+            eu. Quisque augue purus, porttitor quis sagittis vitae, cursus vel
+            turpis.
+          </p>
+        </div>
+
+        <aside className="w-full md:w-[25%] mt-8 md:m-0" aria-label="sidebar">
+          <figure>
+            <div className="relative block w-full h-[300px]">
+              <Image
+                src={muzeum}
+                layout="fill"
+                className="relative block"
+                objectFit="contain"
+              />
+            </div>
+            <figcaption className="text-center text-sm text-dark-300 pb-5">
+              Muzeum trati na nádraží v Loděnici při slavnostním otevření.
+            </figcaption>
+          </figure>
+
+          <div
+            onClick={openDetailsHandler}
+            className="md:hidden font-bold p-3 bg-accent-200 cursor-pointer flex justify-between"
+          >
+            {!isDetailsOpen ? (
+              <h2>Zobrazit detaily</h2>
+            ) : (
+              <h2>Skrýt detaily</h2>
+            )}
```

Comment:
This could be a bit more simple: 

```suggestion
            <h2>{!isDetailsOpen ? 'Zobrazit detaily' : 'Skrýt detaily'}</h2>
```